src/components/atoms/Badge/Badge.md

```diff
@@ -0,0 +1,4 @@
+# # Badge Component
+
```

Comment:
This should be ## not # #

See the issue here:

<img width="1031" alt="Screenshot 2020-03-20 at 3 40 46 PM" src="https://user-images.githubusercontent.com/5359038/77154175-3514a800-6ac1-11ea-9138-83dd73834632.png">